src/Commands.pm

```diff
@@ -4631,7 +4631,10 @@ sub cmdStorage {
 			cmdStorage_log();
 		} elsif ($switch eq 'desc') {
 			cmdStorage_desc($items);
-		} elsif ($switch eq 'add' || $switch eq 'addfromcart' || $switch eq 'get' || $switch eq 'gettocart' || $switch eq 'close') {
+		}elsif(($switch =~ /^(add|addfromcart|get|gettocart)$/ && $items)
```

Comment:
Sigh. Why make the whitespace ugly?

Reply:
lol I agree they are ugly, but at least it is working.